p2panda-rs/src/document/document_view_id.rs

```diff
@@ -34,13 +34,33 @@ pub struct DocumentViewId(Vec<OperationId>);
 impl DocumentViewId {
     /// Create a new document view id.
     pub fn new(graph_tips: Vec<OperationId>) -> Self {
```

Comment:
Should this parameter be a reference `&Vec<OperationId>`?

Reply:
How about `&[OperationId]`?